src/Tests/main.test.ts

```diff
@@ -0,0 +1,65 @@
+import * as core from "@actions/core";
+import { main } from "../main";
+import { AuthorizerFactory } from 'azure-actions-webclient/AuthorizerFactory';
+import { AzureAppServiceUtility } from 'azure-actions-appservice-rest/Utilities/AzureAppServiceUtility';
+import { ContainerDeploymentUtility } from 'azure-actions-appservice-rest/Utilities/ContainerDeploymentUtility';
+import { KuduServiceUtility } from 'azure-actions-appservice-rest/Utilities/KuduServiceUtility';
+import { AzureAppService } from 'azure-actions-appservice-rest/Arm/azure-app-service';
+import { TaskParameters } from "../taskparameters";
+
+jest.mock('@actions/core');
+jest.mock('../taskparameters');
+jest.mock('azure-actions-webclient/AuthorizerFactory');
+jest.mock('azure-actions-appservice-rest/Arm/azure-app-service');
+jest.mock('azure-actions-appservice-rest/Utilities/AzureAppServiceUtility');
+jest.mock('azure-actions-appservice-rest/Utilities/ContainerDeploymentUtility');
+jest.mock('azure-actions-appservice-rest/Utilities/KuduServiceUtility');
+
+describe('main.ts tests', () => {
+
+    afterEach(() => {
+        jest.restoreAllMocks();
+    })
+        
+    it("gets inputs and executes all the functions", async () => {
```

Comment:
Comment what scenario you have tested here. This will set a trend that anyone who adds more tests describes the case and helps in maintaining  and readability

Reply:
It is done